src/main/scala/org/ergoplatform/ErgoApp.scala

```diff
@@ -107,6 +107,9 @@ class ErgoApp(args: Args) extends ScorexLogging {
     ErgoNodeViewSynchronizer(networkControllerRef, nodeViewHolderRef, ErgoSyncInfoMessageSpec,
       ergoSettings, timeProvider)
 
+  val peerSynchronizer: ActorRef = PeerSynchronizerRef("PeerSynchronizer",
```

Comment:
I'm not sure I understand what's going on here. `peerSynchronizer` is not used anywhere. Is it just "started" here? Why is it not private?

Reply:
Made it private, also added the comment:
"Launching PeerSynchronizer actor which is then registering itself at network controller"